### ISSUE TITLE
Integrate the Norwegian, Icelandic and Farsi models 

### ISSUE THREAD

**Issue by andrenatal** [mozilla/firefox-translations#90]
@abhi-agg and @eu9ene what you think?

**Comment by eu9ene** [comment]
sure, why not

**Comment by andrenatal** [comment]
Cool, want to take that one since you've one this before?

**Comment by eu9ene** [comment]
yep, I can add them

**Comment by kpu** [comment]
Farsi is 7 BLEU behind where it should be.  Icelandic and Norwegian were trained for ParaCrawl purposes where the intended use is corpus mining not readable output.  I just want to make clear we're not endorsing the quality of these models, but if operationally you believe some support is better than no support, carry on.

**Comment by eu9ene** [comment]
I've already added them to our storage and completed evaluation. Judging from the numbers I think only nb-en might be considered as a prod quality model. I put everything else to dev. They still might be useful for testing purposes. I added a flag to be able to distinguish prod and dev models on extension level, so that it's easy to disable them or mark as test.

current dev models evaluation: https://github.com/mozilla/firefox-translations-models/blob/0.2.13/evaluation/dev/results.md
current prod models evaluation: https://github.com/mozilla/firefox-translations-models/blob/0.2.13/evaluation/prod/results.md
